tree/10_ncf_internals/list-compatible-inputs.ps1

```diff
@@ -5,24 +5,84 @@ param(
 )
 $path="$framework_path\$nn_directory"
 
-# Create an empty exclude list
+# We consider that the Windows agent doesn't have any capability
+
+$version_regex="([0-9]+).([0-9]+)*"
+$agent_version_regex="^#[\s]*@agent_version[\s]*(>=|<)[\s]*$($version_regex)"
+# capture capabilities, and remainder of line
+$agent_requirements_regex="^#[\s]*@agent_requirements[\s]*""capabilities""[\s]*:[\s]*\[[\s]*(.*)[\s]*\](.*)"
+$reminder_regex="[\s]*(&&|\|\|)[\s]*""agent_version""[\s]*(>=|<)[\s]*$($version_regex)"
```

Comment:
Remainder is what remains.
Reminder is what you put put on your fridge to remind you something.